snekomatic/app.py

```diff
@@ -1,16 +1,12 @@
-import sys
 import os
 import trio
-from glom import glom
 import hypercorn
 import hypercorn.trio
 import quart
 from quart import request
 from quart_trio import QuartTrio
-import gidgethub
 
-from .db import PersistentStringSet
-from .gh import GithubApp
+from .events import github_app
```

Comment:
It's not "events", it's
```suggestion
from .event_handlers import github_app
```
conceptionally.

Reply:
Fair enough

Reply:
I like both of the suggestions! I definitely prefer taking it one step further and splitting up the modules into logically related features. I'm willing to do that for this MR.